redaxo/src/core/pages/packages.list.php

```diff
@@ -6,26 +6,15 @@
 
 echo rex_view::title(rex_i18n::msg('addons'), '');
 
-$content = '';
-
 // the package manager don't know new packages in the addon folder
 // so we need to make them available
 rex_package_manager::synchronizeWithFileSystem();
 
-$toolbar = '
-    <div class="form-group form-group-xs">
-        <div class="input-group input-group-xs" id="rex-js-available-addon-search">
-            <input class="form-control" type="search" placeholder="' . rex_i18n::msg('package_search') . '" '.(rex_request('function') ? '' : 'autofocus ').'/>
```

Comment:
Werden die keys noch genutzt oder können auch weg?

Reply:
Die wurden schon in dem Original-PR entfernt